### DISCUSSION TITLE
Cognition run suddenly stopped working

### DISCUSSION THREAD

**Discussion by LizSmeets** [jspsych/jsPsych#2730] [Q&A]
Hi Everyone,
This is not a jspsych but a cognition.run question, which I assume many of you are using to collect data and there is no separate forum. As of yesterday, all my scripts stopped working on cognition.run, the links in the browser just show a white screen, also the preview shows only a white screen. Does any of you experience the same problem or does anyone know how to start solving this? Thanks!

**Comment by sophief10** [comment]
@LizSmeets I am not experiencing this when using cognition.run, so the problem must have something to do with your browser or computer. Have you tried logging in using a different browser?
